cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,67 @@
-describe('Cypress application', () => {
-  before(() => {
+const alertMassage = {
+  seeAlert: 'You clicked a button',
+  waitAlert: 'This alert appeared after 5 seconds',
+  confirmAlert: 'Do you confirm action?',
+  selectMassage: 'You selected Ok',
+  cencelMassage: 'You selected Cancel',
```

Comment:
There is a typographical error in 'cencelMassage'. It should be 'cancelMessage'. Always ensure that variable names are spelled correctly.